replace_refs.py

```diff
@@ -0,0 +1,27 @@
+#! /usr/bin/env python3
+import argparse
+from argparse import RawTextHelpFormatter
+import linecache
+
+description="replaces the REF allele with the old REF allele if REF = ALT in the remapped VCF"
+
+parser = argparse.ArgumentParser(description = description, formatter_class=RawTextHelpFormatter)
+parser.add_argument("-i", "--vcf", help="VCF file containing remapped variants")
+parser.add_argument("-r", "--oldrefalleles", help="file containing the same header as the VCF file + the old" 
```

Comment:
```suggestion
parser.add_argument("-r", "--oldrefalleles", help="file containing the same header as the VCF file + the old "
```

Space is required, otherwise it will concatenate as "oldREF"